### ISSUE TITLE
admin无法删除用户：当前操作被禁止，请确认你有合法的权限。

### ISSUE THREAD

**Issue by paddy235** [vmware/harbor#3058]
If you are reporting a problem, please make sure the following information are provided:
1)Version of docker engine and docker-compose.
```
[root@dockerreg 2017-08-16]# docker version
Client:
 Version:      17.05.0-ce
 API version:  1.29
 Go version:   go1.7.5
 Git commit:   89658be
 Built:        Thu May  4 22:06:25 2017
 OS/Arch:      linux/amd64

Server:
 Version:      17.05.0-ce
 API version:  1.29 (minimum version 1.12)
 Go version:   go1.7.5
 Git commit:   89658be
 Built:        Thu May  4 22:06:25 2017
 OS/Arch:      linux/amd64
 Experimental: false


[root@dockerreg 2017-08-16]# docker-compose --version
docker-compose version 1.14.0, build c7bdf9e

```
2)Config files of harbor, you can get them by packaging "harbor.cfg" and files in the same directory, including subdirectory.
[配置文件和日志](https://github.com/vmware/harbor/files/1227105/harbor.tar.gz)
3)Log files, you can get them by package the /var/log/harbor/ .
[配置文件和日志](https://github.com/vmware/harbor/files/1227105/harbor.tar.gz)

**Comment by paddy235** [comment]
升级到1.2.0-rc1问题依然存在

**Comment by lazize** [comment]
Sorry but I had to make this comment ... :wink: 
It would be much simple if it was in plain English!

**Comment by reasonerjt** [comment]
@paddy235 
The 2 links you attached are all configuration files, please provide log files so we can investigate.

**Comment by reasonerjt** [comment]
close due to inactivity.
